bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -311,9 +443,77 @@
 		<label>@text/channelPowerLabel</label>
 		<description>@text/channelPowerDescription</description>
 		<category>Number</category>
-		<state readOnly="true" pattern="%.0f %unit%"/>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="energy">
+		<item-type>Number:Energy</item-type>
+		<label>@text/channelEnergyLabel</label>
+		<description>@text/channelEnergyDescription</description>
+		<category>Number</category>
+		<state readOnly="true"/>
```

Comment:
Same on new channel types, a default pattern would be a good thing.

Reply:
Done